src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="container"
+      data-qa="big"
+    >
+      <div class="big-search">
+        <input
+          class="input input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="small-search"
+        data-qa="small"
```

Comment:
The `data-qa` attribute for the small search form is incorrectly assigned as `small`. Please ensure it matches the task requirements.